css/properties/white-space.json

```
@@ -96,6 +96,54 @@
             }
           }
         },
+        "nowrap": {
+          "__compat": {
+            "description": "<code>nowrap</code>",
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": "18"
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "3.5"
```

There's some evidence that this predates Firefox 1 https://bugzilla.mozilla.org/show_bug.cgi?id=45621

```suggestion
                "version_added": "1"
```